### ISSUE TITLE
how to post a video?

### ISSUE THREAD

**Issue by FastPure** [jlobos/instagram-web-api#218]
title

**Comment by joweste** [comment]
I´d like to know as well.

**Comment by MonsieurBibo** [comment]
I don't think it is possible right now with this package. However you can still try with `uploadPhoto()`

**Comment by XeraFiu-YTB** [comment]
I will make a Pull resquests later to add this functionnality.

**Comment by MonsieurBibo** [comment]
@joweste Will you PR this or not ?

**Comment by MonsieurBibo** [comment]
Well my bad I got the wrong person pinged

**Comment by PenguinOfTheSky** [comment]
> However you can still try with uploadPhoto()

99% sure that doesn't work. i tried a few times.
